src/noAttributeParameterDecoratorRule.ts

```diff
@@ -29,9 +29,9 @@ export class Rule extends Lint.Rules.AbstractRule {
             const syntaxKind = SyntaxKind.current();
             return Maybe.lift(node.parent)
                 .fmap(parent => {
-                    if (parent.kind === syntaxKind.ClassExpression) {
+                    if (parent.kind === syntaxKind.ClassExpression && parent.parent.name) {
```

Comment:
parent.parent is always defined ?

Reply:
Yes, the constructor is always in a class.